.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+name: Build
+on:
+  pull_request:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: npm ci
+      - name: Generate Bundle
+        run: npm run build:bundle
+      - name: Compare the expected and actual dist/ directories
+        # We need to make sure the checked-in `index.js` actually matches what we expect it to be.
+        id: diff
+        run: |
+          if [ "$(git diff --ignore-space-at-eol dist/ | wc -l)" -gt "0" ]; then
+            echo "Detected uncommitted changes after build.  See status below:"
+            git diff
+            exit 1
+          fi
+        # If index.js was different than expected, upload the expected version as an artifact
```

Comment:
please move this comment to the item it belongs to.
currently the comment is part of the `name: Compare the expected and actual dist/ directories ...`